java/test3/SpanTest.java

```diff
@@ -0,0 +1,36 @@
+import java.io.IOException;
+import java.util.*;
+import java.lang.RuntimeException;
+
+import ddlogapi.DDlogAPI;
+import ddlogapi.DDlogCommand;
+import ddlogapi.DDlogRecord;
+
+public class SpanTest {
+    private final DDlogAPI api;
+
+    SpanTest() {
+        this.api = new DDlogAPI(1, r -> this.onCommit(r), false);
```

Comment:
Can we pass this as a callback to `commit_dump_changes` instead? I'd like to discourage people from using this callback...